cypress/integration/lesson14_spec.js

```diff
@@ -0,0 +1,34 @@
+describe('My First Test', () => {
+  before(() => {
+    cy.visit('https://demoqa.com')
+    })
+
+    after(() => {
+      cy.reload(); // cleanup text from the inputs fields
```

Comment:
better it add to before each, due to cypress bug

Reply:
Accepted.
Fixed.